ci/prebuilt/BUILD

```diff
@@ -8,6 +8,12 @@ cc_library(
 )
 
 cc_library(
+    name = "backward",
+    hdrs = glob(["thirdparty_build/include/*.hpp"]),
```

Comment:
Best not to glob all the `*.hpp` here, since this might drag in `.hpp` from other libraries. There was a time when `thirdparty_build` had its own subdirectory for each dep, but I eliminated it for simplicity as it wasn't needed anymore. The price we pay is that we need to be more precise here.